cloudflow-enterprise-components/templates/03-cloudflow-enterprise-operator-serviceaccount.yaml

```diff
@@ -0,0 +1,8 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: {{ .Values.serviceAccount }}
```

Comment:
I think you need to change the default name for the service account:
https://github.com/lightbend/cloudflow-helm-charts/blob/2e8c6dfa64a9a7184c06d4afc4530b51cb8f035b/cloudflow-enterprise-components/values.yaml#L1

it might match with the `cloudflow-operator` one, isn't it?

Reply:
https://github.com/lightbend/cloudflow-helm-charts/pull/24/files#diff-e49d6960749258dcd87d4739d6d14745a4083c72e53054d8c51a97228b78d9f6R1